sklearn/metrics/_ranking.py

```diff
@@ -1408,6 +1409,14 @@ def _tie_averaged_dcg(y_true, y_score, discount_cumsum):
     return (ranked * discount_sums).sum()
 
 
+def _check_length_greater_1(y_true: numpy.ndarray):
+    shape = y_true.shape
+    if len(shape) > 1 and shape[1] <= 1:
+        raise ValueError(
+            "Only lengths > 1 are supported. Got length {}".format(shape[1])
```

Comment:
I think that we can relate to what the shape means.

```suggestion
            f"Computing NDCG is only meaningful when there is more than 1 document. "
            "Got {y_true.shape[1]} instead.
```

Reply:
Done. 

I see there is disagreement over whether to use the `format` method or new(ish) style format strings :)

Reply:
Yep, we tend to write new code with newer Python feature but we still have old code that we did modify.

Reply:
@glemaitre What does it mean by "more than 1 document"? Or it's more than 1 dimension and this is a typo?